src/views/Upload.vue

```diff
@@ -4,45 +4,48 @@
 			class="w-screen h-screen absolute bg-primary-gradient clip-half-circle"
 		/>
 		<Navigation class="z-10" />
-		<div class="container mx-auto rounded-lg shadow-xl my-12 bg-white z-10">
-			<div class="px-8 pt-10 pb-2">
-				<h1 class="text-3xl leading-10 font-bold text-gray-900">
-					Kontrola plagiátorstva
-				</h1>
-				<h3 class="text-xl mt-2 leading-8 text-gray-800">
-					Pridajte jednu alebo viac prác, ktoré si prajete
-					skontrolovať. Nahrajte súbory alebo vložte text práce.
-				</h3>
-			</div>
-			<div class="px-20 pb-10 pt-5">
-				<div class="flex">
-					<UploadTab
-						v-for="tab in tabs"
-						:key="tab"
-						:title="tab"
-						:active="tab === selectedTab"
-						@click.native="selectedTab = tab"
-					/>
+		<div class="flex items-center md:container md:mx-auto w-full">
```

Comment:
The top margin on the white table makes the navigation margin unproportional from bottom.
![image](https://user-images.githubusercontent.com/27869979/109662785-b991f680-7b6b-11eb-9f78-9d82454a3032.png)